optionals/bnae_falkor_compat_ace/config.cpp

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"tac_main", "bnae_falkor", "bnae_core"};
```

Comment:
Does `bnae_falkor` not already require `bnae_core`? O.o

Reply:
not sure, that's why I left it as is.